lib/node_modules/@stdlib/array/base/reshape/test/test.js

```diff
@@ -0,0 +1,337 @@
+		],
+		[
+			[ 5, 6, 7, 8 ]
+		]
+	];
+	actual = reshape( x, [ 2, 2, 2 ], [ 2, 1, 4 ], false );
+	t.deepEqual( actual, expected, 'returns expected value' );
+
+	expected = [
+		[
+			[ 1, 2 ],
+			[ 3, 4 ],
+			[ 5, 6 ],
+			[ 7, 8 ]
+		]
+	];
+	actual = reshape( x, [ 2, 2, 2 ], [ 1, 4, 2 ], false );
+	t.deepEqual( actual, expected, 'returns expected value' );
+
+	t.end();
+});
+
+tape( 'the function returns a filled nested array (4d, lexicographic)', function test( t ) {
+	var expected;
+	var actual;
+	var x;
+
+	x = [
+		[
+			[
+				[ 1, 2, 3, 4]
+			],
+			[
+				[ 5, 6, 7, 8 ]
+			]
+		]
+	];
+
+	expected = [
+		[
+			[
+				[ 1, 2 ],
+				[ 3, 4 ]
+			],
+			[
+				[ 5, 6 ],
+				[ 7, 8 ]
+			]
+		]
+	];
+	actual = reshape( x, [ 1, 2, 1, 4 ], [ 1, 2, 2, 2 ], false );
+	t.deepEqual( actual, expected, 'returns expected value' );
+
+	expected = [
+		[
+			[
+				[ 1, 2 ],
+				[ 3, 4 ]
+			]
+		],
+		[
+			[
+				[ 5, 6 ],
+				[ 7, 8 ]
+			]
+		]
+	];
+	actual = reshape( x, [ 1, 2, 1, 4 ], [ 2, 1, 2, 2 ], false );
+	t.deepEqual( actual, expected, 'returns expected value' );
+
+	expected = [
+		[
+			[
+				[ 1 ],
+				[ 2 ]
+			],
+			[
+				[ 3 ],
+				[ 4 ]
+			]
+		],
+		[
+			[
+				[ 5 ],
+				[ 6 ]
+			],
+			[
+				[ 7 ],
+				[ 8 ]
+			]
+		]
+	];
+	actual = reshape( x, [ 1, 2, 1, 4 ], [ 2, 2, 2, 1 ], false );
+	t.deepEqual( actual, expected, 'returns expected value' );
+
+	t.end();
+});
+
+tape( 'the function returns a reshaped nested array (2d, colexicographic)', function test( t ) {
+	var expected;
+	var actual;
+	var x;
+
+	x = [ [ 1, 2, 3 ], [ 4, 5, 6 ] ];
+
+	expected = [ [ 1, 4 ], [ 2, 5 ], [ 3, 6 ] ];
+	actual = reshape( x, [ 2, 3 ], [ 3, 2 ], true );
+	t.deepEqual( actual, expected, 'returns expected value' );
+
+	expected = [ [ 1, 4, 2, 5, 3, 6 ] ];
+	actual = reshape( x, [ 2, 3 ], [ 1, 6 ], true );
+	t.deepEqual( actual, expected, 'returns expected value' );
+
+	t.end();
+});
+
+tape( 'the function returns a reshaped nested array (3d, colexicographic)', function test( t ) {
+	var expected;
+	var actual;
+	var x;
+
+	x = [
+		[
+			[ 1, 2 ],
+			[ 3, 4 ]
+		],
+		[
+			[ 5, 6 ],
+			[ 7, 8 ]
+		]
+	];
+
+	expected = [
+		[
+			[ 1, 5 ]
+		],
+		[
+			[ 3, 7 ]
+		],
+		[
+			[ 2, 6 ]
+		],
+		[
+			[ 4, 8 ]
+		]
+	];
+	actual = reshape( x, [ 2, 2, 2 ], [ 4, 1, 2 ], true );
+	t.deepEqual( actual, expected, 'returns expected value' );
+
+	expected = [
+		[
+			[ 1, 5, 3, 7 ]
+		],
+		[
+			[ 2, 6, 4, 8 ]
+		]
+	];
+	actual = reshape( x, [ 2, 2, 2 ], [ 2, 1, 4 ], true );
+	t.deepEqual( actual, expected, 'returns expected value' );
+
+	expected = [
+		[
+			[ 1, 5 ],
+			[ 3, 7 ],
+			[ 2, 6 ],
+			[ 4, 8 ]
+		]
+	];
+	actual = reshape( x, [ 2, 2, 2 ], [ 1, 4, 2 ], true );
+	t.deepEqual( actual, expected, 'returns expected value' );
+
+	t.end();
+});
+
+tape( 'the function returns a filled nested array (4d, colexicographic)', function test( t ) {
+	var expected;
+	var actual;
+	var x;
+
+	x = [
+		[
+			[
+				[ 1, 2, 3, 4]
```

Comment:
```suggestion
				[ 1, 2, 3, 4 ]
```

Use consistent spacing.